packages/react/src/components/Storage/FileUploader/Previewer/index.tsx

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
Can we just remove the unused variables?

Reply:
Sure